frontend/web/components/modals/CreateFlag.js

```diff
@@ -187,6 +198,50 @@ const CreateFlag = class extends Component {
     ) {
       this.getFeatureUsage()
     }
+
+    if (
+      this.props.projectFlag &&
+      Utils.getFlagsmithHasFeature('github_integration')
+    ) {
+      getExternalResource(getStore(), {
+        feature_id: this.props.projectFlag.id,
+      }).then((res) => {
+        this.setState({ externalResources: res.data.results })
```

Comment:
We'll be able to remove state in tsx components with hooks

Reply:
Was removed, and a new component was added with hooks.